packages/runtime/src/next/api-handler.ts

```diff
@@ -100,7 +100,7 @@ export function MakeswiftApiHandler(
           return res.status(400).json({ message: 'Bad Request' })
         }
 
-        const revalidate = res.revalidate ?? res.unstable_revalidate
+        const revalidate = res.revalidate
```

Comment:
This change should come with a changeset that clearly explains what's happening and communicates this as a breaking change.

Reply:
Added the changeset. We can review the final changeset/release notes later once we want to publish the new version.